x-pack/plugins/siem/public/components/help_menu/help_menu.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiButton, EuiHorizontalRule, EuiIcon, EuiTitle, EuiSpacer } from '@elastic/eui';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import styled from 'styled-components';
+
+export const Icon = styled(EuiIcon)`
+  margin-right: ${theme.euiSizeS};
+`;
+
+export class HelpMenuComponent extends React.PureComponent {
+  public render() {
+    return (
+      <>
+        <EuiHorizontalRule margin="none" />
+
+        <EuiSpacer />
+
+        <EuiTitle size="xxs">
+          <h6>
+            <Icon type="securityAnalyticsApp" />
+            SIEM Application Help
```

Comment:
Do we want to `i18n` this text?

Reply:
Ah, I always forget that part. Adding now.